app/config/errors.php

```diff
@@ -0,0 +1,442 @@
+        'name' => Exception::ATTRIBUTE_NOT_FOUND,
+        'description' => 'Attribute with the requested ID could not be found.',
+        'code' => 404,
+    ],
+    Exception::ATTRIBUTE_UNKNOWN => [
+        'name' => Exception::ATTRIBUTE_UNKNOWN,
+        'description' => 'The attribute required for the index could not be found. Please confirm all your attributes are in the <span class="tag">available</span> state.',
+        'code' => 400,
+    ],
+    Exception::ATTRIBUTE_NOT_AVAILABLE => [
+        'name' => Exception::ATTRIBUTE_NOT_AVAILABLE,
+        'description' => 'The requested attribute is not yet <span class="tag">available</span>. Please try again later.',
+        'code' => 400,
+    ],
+    Exception::ATTRIBUTE_FORMAT_UNSUPPORTED => [
+        'name' => Exception::ATTRIBUTE_FORMAT_UNSUPPORTED,
+        'description' => 'The requested attribute format is not supported.',
+        'code' => 400,
+    ],
+    Exception::ATTRIBUTE_DEFAULT_UNSUPPORTED => [
+        'name' => Exception::ATTRIBUTE_DEFAULT_UNSUPPORTED,
+        'description' => 'Default values cannot be set for <span class="tag">array</span> and <span class="tag">required</span> attributes.',
+        'code' => 400,
+    ],
+    Exception::ATTRIBUTE_ALREADY_EXISTS => [
+        'name' => Exception::ATTRIBUTE_ALREADY_EXISTS,
+        'description' => 'Attribute with the requested ID already exists.',
+        'code' => 409,
+    ],
+    Exception::ATTRIBUTE_LIMIT_EXCEEDED => [
+        'name' => Exception::ATTRIBUTE_LIMIT_EXCEEDED,
+        'description' => 'The maximum number of attributes has been reached.',
+        'code' => 400,
+    ],
+    Exception::ATTRIBUTE_VALUE_INVALID => [
+        'name' => Exception::ATTRIBUTE_VALUE_INVALID,
+        'description' => 'The attribute value is invalid. Please check the type, range and value of the attribute.',
+        'code' => 400,
+    ],
+
+    /** Indexes */
+    Exception::INDEX_NOT_FOUND => [
+        'name' => Exception::INDEX_NOT_FOUND,
+        'description' => 'Index with the requested could not be found.',
+        'code' => 404,
+    ],
+    Exception::INDEX_LIMIT_EXCEEDED => [
+        'name' => Exception::INDEX_LIMIT_EXCEEDED,
+        'description' => 'The maximum number of indexes has been reached.',
+        'code' => 400,
+    ],
+    Exception::INDEX_ALREADY_EXISTS => [
+        'name' => Exception::INDEX_ALREADY_EXISTS,
+        'description' => 'Index with the requested ID already exists.',
+        'code' => 409,
+    ],
+
+    /** Query */
+    Exception::QUERY_LIMIT_EXCEEDED => [
+        'name' => Exception::QUERY_LIMIT_EXCEEDED,
+        'description' => 'Query limit exceeded for the current attribute. Usage of more than 100 query values on a single attribute is prohibited.',
+        'code' => 400,
+    ],
+    Exception::QUERY_INVALID => [
+        'name' => Exception::QUERY_INVALID,
+        'description' => 'Your query syntax is invalid. Please check your query and try again.',
```

Comment:
I see. Thanks for pointing that out. But @eldadfux  and I talked about moving this over under the general section since `Query` aren't really Appwrite entities and does not sit well with the naming convention we have outlined above.

Reply:
Makes sense.